signal_scanner_bot/transport.py

```diff
@@ -1,72 +1,108 @@
 """Main module."""
+import pprint
 import asyncio
 import logging
 import subprocess
 import concurrent.futures
-import urllib3
 
-import tweepy
+import peony
 import ujson
 
 from . import env
 from . import messages
-from . import twitter
 from . import signal
 
 log = logging.getLogger(__name__)
 
 
 ################################################################################
-# Stream Listener
+# Twitter-to-Queue
 ################################################################################
-class Listener(tweepy.StreamListener):
-    """
-    Listener implementation for when a status matching the track is received.
-    """
-
-    def on_status(self, status: tweepy.Status):
-        if not env.STATE.LISTENING:
-            return
-
-        messages.process_twitter_message(status)
+def _filter_hashtags(data, filter_hashtag_list):
+    for input_hashtag in data["entities"]["hashtags"]:
+        if input_hashtag["text"] in filter_hashtag_list:
+            return True
+    return False
```

Comment:
Filter to check if a tweet from a user contains any of the listed hashtags to monitor. Something new to note regarding this is that the hashtags will either be listed without the "#" in the .env file or we will need to strip the "#" from the config files. The hashtag entity in the status object is only the text and doesn't actually include the "#" in it.